app/main.py

```diff
@@ -14,4 +14,11 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    # Convert both the phrase and the letter to lowercase
```

Comment:
please, remove the comments

your code should be self-descriptive so no need to use comments